### PR TITLE
Feature/grails5

### PR DESCRIPTION
**Pull request by sughics**
Combined Grails4, Grails5 upgrade. 

Deployment running on https://sds-test.ala.org.au with cognito authentication.

### CODE REVIEW

File: grails-app/conf/logback.xml
Comment (charvolant) [inline grails-app/conf/logback.xml:1]: Does this need a production configuration that will log to an sds.log file the /var/log/tomcat9 directory or is that taken care of elsewhere?

File: grails-app/controllers/au/org/ala/sds/SDSController.groovy [hunk 1]
```diff
@@ -71,6 +106,7 @@ class SDSController {
     /**
      * SDS layers
      */
+
```
Comment (charvolant) [inline grails-app/controllers/au/org/ala/sds/SDSController.groovy]: Add an @Operation here too?

**Review by charvolant — APPROVED** [review]

File: grails-app/conf/logback.xml
Comment (sughics) [reply grails-app/conf/logback.xml:1]: hi @charvolant , the logback file and config for prod usage is handled on the ansible side: 
 - https://github.com/AtlasOfLivingAustralia/ala-install/tree/feature/grails5-updates/ansible/roles/sds/templates
 - https://github.com/AtlasOfLivingAustralia/ansible-inventories/blob/ae9a1d6d52f4dd3df1dd47a719b0144a9254b5e0/sds/sds-prod-2022#L32

File: grails-app/controllers/au/org/ala/sds/SDSController.groovy (hunk 1 above)
Comment (sughics) [reply grails-app/controllers/au/org/ala/sds/SDSController.groovy]: Added. And also included in API Gateway Specs for sds-webapp APIs: https://github.com/AtlasOfLivingAustralia/aws-apig-cognito/blob/test/sds-webapp/app_setup/apis/app-rest.yaml  -> https://api.test.ala.org.au/sds-webapp/ws/layers

**Pull request merged**